cg/meta/archive/archive.py

```diff
@@ -103,12 +103,21 @@ def archive_all_non_archived_spring_files(
                 archive_location=archive_location,
             )
 
-    def retrieve_samples(self, sample_internal_ids: list[str]) -> None:
-        """Retrieves the archived spring files for a list of samples."""
-        samples: list[Sample] = [
-            self.status_db.get_sample_by_internal_id(sample_internal_id)
-            for sample_internal_id in sample_internal_ids
+    def retrieve_case(self, case_id: str):
+        case = self.status_db.get_case_by_internal_id(case_id)
+        self.retrieve_samples(self.get_archived_samples(case))
+
+    def get_archived_samples(self, case: Case):
```

Comment:
Add return type hint

Reply:
If add tags as an arg then the function name is fine otherwise it should mention spring